apps/music/style/music.css

```diff
@@ -585,10 +585,12 @@ body.picker-mode #title-player {
 
 #player-cover.slideOut #player-cover-caption {
   transform: translateY(100%);
+  visibility: visible;
```

Comment:
This visibility change looks good but just wondering if it's necessary or an improvement for the css styles.